streamz/sources.py

```diff
@@ -597,27 +597,32 @@ def from_kafka_batched(topic, consumer_params, poll_interval='1s',
         If engine is set to "cudf", streamz reads data (messages must be JSON) from Kafka
         in an accelerated manner directly into cuDF (GPU) dataframes.
 
+        Please refer to RAPIDS cudf API here:
+        https://docs.rapids.ai/api/cudf/stable/
+
         This is done using the custreamz.kafka module in cudf. custreamz.kafka has the exact
-        same API as Confluent Kafka, so it serves as a drop-in replacement in from_kafka_batched
-        with minimal duplication of code. But under the hood, it reads messages from librdkafka
-        and directly uploads them to the GPU as a cuDF dataframe instead of gathering all the
-        messages back from C++ into Python. This essentially avoids the GIL issue described in the
-        Confluent Kafka consumer: https://github.com/confluentinc/confluent-kafka-python/issues/597,
-        and hence enables reading from Kafka in a faster fashion with fewer processes. This
-        accelerated reader also adheres to the checkpointing mechanism in streamz.
+        same API as Confluent Kafka, so it serves as a drop-in replacement with minimal duplication
+        of code. But under the hood, it reads messages from librdkafka and directly uploads them to
+        the GPU as a cuDF dataframe instead of gathering all the messages back from C++ into Python.
+        This essentially avoids the GIL issue described in the Confluent Kafka consumer:
+        https://github.com/confluentinc/confluent-kafka-python/issues/597, and hence enables reading
+        from Kafka in a faster fashion with fewer processes. This accelerated reader also adheres to
+        the checkpointing mechanism in streamz.
 
         Folks interested in trying out custreamz would benefit from this accelerated Kafka reader.
         If one does not want to use GPUs, they can use streamz as is, with the default engine=None.
 
-        To use this option, one must install RAPIDS custreamz using:
-        https://anaconda.org/rapidsai-nightly/custreamz
-        This will install all GPU dependencies, including streamz.
+        To use this option, one must install custreamz (use the appropriate CUDA version recipe) using
+        the following command, which will install all GPU dependencies, including streamz:
+        conda env create --name custreamz --file
```

Comment:
Seems strange to require the user to create a new environment; also, this should only work with the raw version of the file, not the rendered github page

Reply:
> Seems strange to require the user to create a new environment

That's true, but the alternative is to install quite a few packages instead. The .yml script installs everything cleanly. Also, this is temporary (as mentioned at the end of the doc string) until the code gets merged into cudf/custreamz, after which it be a single line nightly package install. 

> this should only work with the raw version of the file, not the rendered github page

Correct, I was assuming people would understand this by themselves, that they would need to download the .yml file and then run the command pointing it to that file.

Reply:
Perhaps better to write this out in full, even if it is only temporary

Reply:
Done